Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino

```diff
@@ -235,25 +239,25 @@ void setup() {
   last_encoder_read_time = current_time;
 
   // Initialize ros debug publishers
-#ifdef ENABLE_BUMPER_DEBUG
-  ros_nh.advertise(ros_bumper_debug_pub);
-  timer.every(500, doBumperDebug);
-#endif
-#ifdef ENABLE_ENCODER_DEBUG
-  ros_nh.advertise(ros_encoder_debug_pub);
-  timer.every(500, doEncoderDebug);
-#endif
-#ifdef ENABLE_IMU_DEBUG
-  ros_nh.advertise(ros_imu_debug_pub);
-  timer.every(500, doImuDebug);
-#endif
-#ifdef ENABLE_MAGNETOMETER_DEBUG
-  ros_nh.advertise(ros_magnetometer_debug_pub);
-  timer.every(500, doMagnetometerDebug);
-#endif
-#ifdef ENABLE_TEENSY_DEBUG
-  ros_nh.advertise(ros_teensy_debug_pub);
-#endif
+  if (publish_bumper_debug_info_freq_hz != 0) {
```

Comment:
I was using zero as an indicator that the info is not wanted, so it would not advertise the publisher and not bother with the timer. I figured that having a few less topics hangin around, especially if nothing is actually publishing, would be a good thing. I could do the same with the others, but they are more fundamental.